Src/Eiffel/interface/new_graphical/text_window/text/autocomplete/eb_class_info_analyzer.e

```diff
@@ -435,6 +435,83 @@ feature {NONE}-- Clickable/Editable implementation
 			end
 		end
 
+	described_local (token: EDITOR_TOKEN; line: EDITOR_LINE; ft: FEATURE_AS; ft_start_token: detachable EDITOR_TOKEN_FEATURE_START): detachable TUPLE [feat: E_FEATURE; ast: detachable AST_EIFFEL]
+			-- search in feature represented by `ft' the feature associated with `token' if any
```

Comment:
I am not sure what you mean with same order as the compiler .. and where to look for in AST_FEATURE_CHECKER_GENERATOR